gazebo/msgs/performance_metrics.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto2";
+package gazebo.msgs;
+
+/// \ingroup gazebo_msgs
+/// \interface PerformanceMetrics
+/// \brief A message for run-time simulation performance metrics
+
+message PerformanceMetrics
+{
+
+  message PerformanceSensorMetrics
+  {
+    required string sensor_name             = 1;
+    required double real_sensor_update_rate = 2;
+    required double sim_sensor_update_rate  = 3;
+    optional double fps                     = 4;
```

Comment:
This is FPS against the simulation clock, yes, and not wall-clock?  Is it instantaneous FPS (for each tick) or an average since time=0?